### PR TITLE
Add setup testing

### PR DESCRIPTION
**Pull request by bam241**
this add a job to ensure package installed properly and can be imported from outside of the repo

### CODE REVIEW

File: python_ci.yml [hunk 1]
```diff
@@ -1,26 +1,52 @@
...
 
 workflows:
     my_workflows:
         jobs:
-            - run_test_py3
-            - run_test_py2
+            - install_py3
+            - run_test_py3:
+                - requires:
+                    install_py3
```
Comment (gonuke) [inline python_ci.yml]: I strongly disagree about this dependency. If anything the dependency should be the other way around.

Fundamental testing should be against the version that is in the repo and NOT against the version that is installed.  CI is testing the PR to the repo.  If something goes wrong (without failing) in the installation, you could get false positives or negatives.

Arguably, there should be no attempt to install until we are sure the tests have passed on the code in the repo.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for putting something about installation back in.  We should probably talk over these dependencies more.

File: python_ci.yml (hunk 1 above)
Comment (bam241) [reply python_ci.yml]: I completely agree with your statement!

I guess I thought about the standard way to do things... install and then test but in this case it does not make sense!

Will fix this tomorrow

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**